src/modules/home/components/accommodation-card.tsx

```diff
@@ -52,32 +53,40 @@ const styles = {
   ratingStar: { fontSize: '17px' },
 };
 
-export function AccommodationCard({ pricePerNight, address, name, rating, image }: AccommodationCardProps) {
+export function AccommodationCard({ id, pricePerNight, address, name, rating, image }: AccommodationCardProps) {
   const [addToWishlist, setAddToWishlist] = useState(true);
+
+  const handleHeartClick = (event: React.MouseEvent) => {
+    event.preventDefault();
```

Comment:
If use type = "button" on the component, no need to prevent default it

Reply:
> If use type = "button" on the component, no need to prevent default it

I did do ```type="button"```, but it was still navigating to the given link when heart is pressed if I don't prevent default